src/Traits/Categories/ButtonHelpers.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+use NotificationChannels\OneSignal\OneSignalButton;
+use NotificationChannels\OneSignal\OneSignalWebButton;
+
+trait ButtonHelpers
+{
+
+    /**
+     * Add a web button to the message.
+     *
+     * @param OneSignalWebButton $button
+     *
+     * @deprecated use setWebButton instead
+     *
+     * @return $this
+     */
+    public function webButton(OneSignalWebButton $button)
+    {
+        return $this->setWebButton($button);
+    }
+
+    /**
+     * Adds more than one web button to the message.
+     *
+     * @param array[OnSignalWebButton] $buttons
+     *
+     * @deprecated use setWebButtons instead
+     *
+     * @return $this
+     */
+    public function webButtons(array $buttons)
+    {
+        return $this->setWebButtons($buttons);
+    }
+
+    /**
+     * Add a web button to the message.
+     *
+     * @param OneSignalWebButton $button
+     *
+     * @return $this
+     */
+    public function setWebButton(OneSignalWebButton $button)
+    {
+        return $this->setParameter('web_buttons', array_merge($this->getParameter('web_buttons', []), [$button->toArray()]));
+    }
+
+    /**
+     * Adds more than one web button to the message.
+     *
+     * @param array[OnSignalWebButton] $buttons
+     *
+     * @return $this
+     */
+    public function setWebButtons(array $buttons)
+    {
+        return $this->setParameter('web_buttons', collect($buttons)->map(function ($button) {
```

Comment:
This is probably fine - but you could just make use of the existing method here:

```php
collect($buttons)->each(function ($button) {
    $this->setWebButton($button);
});
```
That way if the key every changes you only have one place to manage it - but again, probably not needed - just an idea :)

Could also apply to `setButtons`?